bot/client.py

```diff
@@ -156,36 +171,55 @@ async def volume(self, context, *, volume: int):
         context.voice_state.volume = volume / 100
         await context.send(f'Volume of the player set to {volume}%')
 
-    @commands.command(aliases=['current', 'playing'])
+    @commands.hybrid_command(
+        name='now',
+        with_app_command=True,
+        description='Show sound currently playing.'
+    )
     async def now(self, context):
         await context.send(embed=context.voice_state.current.create_embed())
 
-    @commands.command()
+    @commands.hybrid_command(
+        name='pause',
+        with_app_command=True,
+        description='Pause my instants sound.'
+    )
     async def pause(self, context):
         if (
-                not context.voice_state.is_playing
-                and context.voice_state.voice.is_playing()
+            not context.voice_state.is_playing
+            and context.voice_state.voice.is_playing()
         ):
+            await context.send('Pausing current sound.')
```

Comment:
The add reaction was the primary feedback tool, but it's incompatible with hybrid_commands.
I'll merge this for now, but later we need to move to slash commands fully
![image](https://user-images.githubusercontent.com/3418974/229375367-71879508-3306-407d-8021-b14a1f701f9f.png)


Can you also remove all the `add_reaction` calls, please?

Reply:
Removed `add_reaction` calls ✅ 

And I really like hybrid commands, it is faster than using slash